tests/system/test_fim/test_synchronization/data/agent_initializing_synchronization.yaml

```diff
@@ -0,0 +1,4 @@
+wazuh-agent1:
+  - regex: .*Finished FIM sync.
```

Comment:
Because there is an initial sync when the agent starts. Checking for the sync to finish, we can now be sure the agent has started properly, an start the rest of the test.